models/sessions.go

```diff
@@ -61,7 +61,7 @@ type Session struct {
 	UpdatedAt time.Time  `json:"updated_at" db:"updated_at"`
 	FactorID  *uuid.UUID `json:"factor_id" db:"factor_id"`
 	AMRClaims []AMRClaim `json:"amr,omitempty" has_many:"amr_claims"`
-	AAL       string     `json:"aal" db:"aal"`
+	AAL       *string    `json:"aal" db:"aal"`
```

Comment:
You can also add a method to `Session` that does the indirection properly. Example:

```go
func (s *Session) GetAAL() string {
  if s.AAL == nil {
    return ""
  }
  return *(s.AAL)
}
```

Reply:
Gotcha, will switch yo using that instead -- think that'd be cleaner